types/picomatch/index.d.ts

```diff
@@ -4,88 +4,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 3.0
 
-import parse = require('./parse');
-import constants = require('./constants');
+import picomatch = require('./lib/picomatch');
```

Comment:
Why do we need everything in a lib folder?

Reply:
I moved them there per [Piotr's comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51523#issuecomment-803549366) because the source files in picomatch itself are [in the lib folder](https://github.com/micromatch/picomatch/tree/master/lib).